crates/aitia/CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 ---
+semver_increment_mode: patch
```

Comment:
What's with all these? Some kind of bad merge?

Reply:
Release automation check was failing so this was @jost-s's resolution. See https://chat.holochain.org/holo/pl/bz4zcsgqk7b1tbgztdcc6zhszh